lib/utils/MyLocalizations.dart

```diff
@@ -6,10 +6,17 @@ import 'package:flutter/services.dart';
 class MyLocalizations {
   Locale locale;
   static Map<dynamic, dynamic>? _localisedValues;
+  static Map<dynamic, dynamic>? _englishValues;
```

Comment:
I personally think that it makes more sense to init this variables as empty dict, it simplify the code after (see translate method)

```dart
static Map<dynamic, dynamic> _localisedValues = {};
static Map<dynamic, dynamic> _englishValues = {};
```